### PR TITLE
Refactor/inspector

### PR DESCRIPTION
**Pull request by peterblazejewicz**
This PR introduces some changes in Inspector page prior to other major task: #330 
- the page relevant styles are moved to `_inspector.scss` partial - so now `index.scss` is Inspector `main` style sheet file:
http://sass-guidelin.es/#main-file
- The `actions` element were a semantic noise within a page - so they are refactored into inlined divs while retaining visual style. They are now transparent to readers:

(do you see them on above screen? No?)

- The main image has no text alternate at all - now it sports with descriptive text (feel free to let me now if it should be fixed/changed):

I'll later add this text as part of readers/crawlers optimization for Inspector indexing

- hide `Chrome.png` from readers/crawlers - this is an image with secondary purpose being part of UI element - so we use empty alt as per specs to hide it. Compare screens:

- there is a CTA - unfortunately written as two separate paragraphs:

It could be (and certainly is) interpreted as two distinct phrases - not good for us. So PR rewrites semantics and style to make it much better while retaining original design:

/cc @jdaudier LG?

Thanks!

Next, task to avoid duplicate words stacked in sections:

(why is this important?)

### CODE REVIEW

**Comment by peterblazejewicz** [comment]
tested with critical for regressions (I think none)

**Comment by jdaudier** [comment]
@peterblazejewicz LGTM :+1: !

**To help with your next task of avoiding duplicate words stacked in sections, here are some copy ideas...**

**OLD**: Understand how your app works without needing to understand how all the code works.  See it in action.

**NEW**: Get an overview of how your app works without needing to understand all the code.  See it in action.
****
**OLD**: Visualize the view hierarchy with the UI tree, see your app activity and history.

**NEW**: See the view hierarchy with the UI tree along with the app activity and history.
****

**OLD**: Inspect your view, UI, events, listener, models, attributes, and print them to the console as variables with one click!

**NEW**: Explore your views, UI, events, listeners, models, attributes, and print them to the console as variables with one click!
****
**OLD**: Explore your radio channels, requests, commands, and app with the Inspector glass.

**NEW**: Use the handy Inspector glass to dive into your radio channels, requests, commands, and app.
****
I think we should change this header too to match the verb tense / style of the other headers.

**OLD**: Customizable
**NEW**: Customize

**Comment by peterblazejewicz** [comment]
Thanks Joanne!
rebase/updated `TodoMvc` > `TodoMVC`. Merging

File: src/stylesheets/pages/_inspector.scss [hunk 1]
```diff
@@ -1,2 +1,209 @@
...
+  img {
+    width: 100%;
+  }
+}
+
+.browser-header {
+  height: 29px;
+  width: 100%;
+  background: #dfe3e6; /* Old browsers */
+  background: -moz-linear-gradient(top,  #dfe3e6 0%, #e0e1e3 8%, #d9dadc 73%, #d7dbde 85%, #d7d8da 100%); /* FF3.6+ */
```
Comment (samccone) [inline src/stylesheets/pages/_inspector.scss]: wont autoprefixer do this for us?

File: src/stylesheets/pages/_inspector.scss [hunk 2]
```diff
@@ -1,2 +1,209 @@
...
+
+  .actions {
+    float: left;
+    margin-left: 5px;
+    display: inline-block;
+
+    .close, .min, .max {
+      float: left;
+      display: inline-block;
+      height: 10px;
```
Comment (samccone) [inline src/stylesheets/pages/_inspector.scss]: `@include size(10px);`

https://github.com/mojotech/mo#size

heheh

File: src/stylesheets/pages/_inspector.scss (hunk 1 above)
Comment (peterblazejewicz) [inline src/stylesheets/pages/_inspector.scss]: updated in: https://github.com/peterblazejewicz/marionettejs.com/commit/07acb278d2415917d984897bd0e623fb8dd064e3
I've left <=IE9 filter thing. As we are using two last versions in autoprefixer we won't get any vendors here.

File: src/stylesheets/pages/_inspector.scss (hunk 2 above)
Comment (peterblazejewicz) [inline src/stylesheets/pages/_inspector.scss]: changed:
https://github.com/peterblazejewicz/marionettejs.com/commit/bc20ed9570803a0552d8ec029bfc45311304e1b3

File: src/stylesheets/pages/_inspector.scss (hunk 1 above)
Comment (samccone) [inline src/stylesheets/pages/_inspector.scss]: wooooord

**Comment by samccone** [comment]
ha review ninja?

**Comment by jdaudier** [comment]
oops sorry... i was playing w/ review ninja today... had no idea how it took over

**Pull request merged**